schemas/host-config.json

```diff
@@ -669,12 +658,12 @@
 				}
 			}
 		},
-		"TextStylesConfig": {
+		"TextBlockConfig": {
```

Comment:
>TextBlockConfig [](http://example.com/codeflow?start=3&length=15)

This should be TextStylesConfig, and should include a TextStyleConfig for each of "heading" and "columnHeader"

TextBlock config should also exist, it contains only "headingLevel"

(per shared model code)